README.md

```diff
@@ -1,5 +1,12 @@
+импорты common.js, работает на node.js
+
+чтобы посмотреть, как все работает, можно зайти в test.js (там есть примеры вызовов и тд)
+
+в папке api - вспомогательные вещи, основные классы - в общей папке
```

Comment:
What you have in `api` folder actually is not related to API, some abstract classes or interfaces like `Product` could be placed in something like `/models/`, auxiliary things like`calculateSmth` are commonly stored in `/utils/` or `/services/` folder

I don't insist on resctructuring all the files right now, anyway it's good that you're trying to do it, but it's just a recommendation for future